libs/eslint-plugin-vx/src/rules/gts_spread_like_types.ts

```diff
@@ -84,8 +82,9 @@ const rule: TSESLint.RuleModule<
         }
 
         switch (node.parent.type) {
-          case AST_NODE_TYPES.CallExpression:
+          /* @ts-expect-error - this is broken because of a regression, fixed by https://github.com/typescript-eslint/typescript-eslint/pull/10024 */
```

Comment:
This PR was merged pretty quickly, but there isn't a release as of this writing.